### PR TITLE
Merge release/2.1 back to dev

### PR DESCRIPTION
**Pull request by HaoK**

### CODE REVIEW

File: migrations/updateMigrations.cmd [hunk 1]
```diff
@@ -1,18 +1,64 @@
+SET version=2.1.0
```
Comment (natemcmaster) [inline migrations/updateMigrations.cmd]: 2.2.0?

File: src/Microsoft.DotNet.Web.ProjectTemplates/content/StarterWeb-CSharp/Data/SqlServer/ApplicationDbContextModelSnapshot.cs [hunk 2]
```diff
@@ -16,7 +15,8 @@ protected override void BuildModel(ModelBuilder modelBuilder)
         {
 #pragma warning disable 612, 618
             modelBuilder
-                .HasAnnotation("ProductVersion", "2.1.0-preview1-28193")
+                .HasAnnotation("ProductVersion", "2.1.0")
```
Comment (natemcmaster) [inline src/Microsoft.DotNet.Web.ProjectTemplates/content/StarterWeb-CSharp/Data/SqlServer/ApplicationDbContextModelSnapshot.cs]: Should this be updated to match what's building on dev?

Comment (HaoK) [reply src/Microsoft.DotNet.Web.ProjectTemplates/content/StarterWeb-CSharp/Data/SqlServer/ApplicationDbContextModelSnapshot.cs]: Sure I was going to do it in a PR after merging, but I can do this as part of the merge too, i'll update this PR

**Comment by HaoK** [comment]
Ok more powershell hacking: --no-restore added, then matched the version from the .csproj to add to the App version, which makes migration run again...  look reasonable @natemcmaster ?

It works now at least :)

**Comment by natemcmaster** [comment]
Looks like a bunch of tests are failing...

**Comment by HaoK** [comment]
Also, doesn't look like appveyor/travis has ever passed one of my prs in this repo :) https://github.com/aspnet/templating/pulls?q=is%3Apr+author%3AHaoK+is%3Aclosed

**Comment by natemcmaster** [comment]
@mkArtakMSFT is anyone assigned to get template tests actually working? I think every PR I've ever reviewed for this repo has failing PR checks.

**Comment by HaoK** [comment]
@kichalla if you are trying to merge templates back to dev you can take over this PR/add more changes to this one

**Comment by natemcmaster** [comment]
@HaoK any status update on this? I need to bring back a few more changes from release/2.1

**Comment by HaoK** [comment]
I think this is good to go, and we can just ignore the failures, do you want to just add the other changes to this branch or do you want me to just squash/merge this PR?

**Comment by natemcmaster** [comment]
Go ahead and merge this in, and I'll merge my changes back separately.

**Pull request merged**